server/build.gradle

```diff
@@ -13,6 +13,12 @@ plugins {
   id 'jacoco'
 }
 
+java {
```

Comment:
This is the new toolchain mechanism for specifying which version of Java we want to use.